app/services/user_alerts/alert_user_about_account_verified.rb

```diff
@@ -2,11 +2,12 @@
 
 module UserAlerts
   class AlertUserAboutAccountVerified
-    def self.call(user:, date_time:, sp_name:)
-      sp_name ||= APP_NAME
+    def self.call(profile:)
+      user = profile.user
+      sp_name ||= profile.initiating_service_provider&.friendly_name || APP_NAME
```

Comment:
this is the first assignment of the `sp_name` local so we can probably remove the `||`
```suggestion
      sp_name = profile.initiating_service_provider&.friendly_name || APP_NAME
```